pkg/network/ebpf/c/protocols/classification/tracer-maps.h

```diff
@@ -1,6 +1,10 @@
 #ifndef __PROTOCOL_CLASSIFICATION_TRACER_MAPS_H
 #define __PROTOCOL_CLASSIFICATION_TRACER_MAPS_H
 
+#ifdef USM_C
+#error "tracer-maps.h should not be included from usm.c"
+#endif
```

Comment:
While that sounds reasonable, can you share the reasons?
if that's a problem of "name collision" we can of course change our file name

Reply:
Its not, usm.o fails to load if the maps are included here.

Reply:
I would say it's the normal behavior to fail to load if you declare map that are not defined in the loader
why having a special check ?

shall we merge the remaining tracer-maps.h into `pkg/network/ebpf/c/tracer` ?
and s|protocols/classification/tracer-maps.h|protocols/classification/shared-tracer-maps.h|g for `pkg/network/ebpf/c/tracer/stats.h` `pkg/network/ebpf/c/tracer/events.h`